docs/tools/terraform/howto/config-postgresql-provider.rst

```diff
@@ -0,0 +1,80 @@
+Use PostgreSQL® provider with Aiven's PostgreSQL service
+#####################################################
+
+Aiven's Terraform provider is aimed for provisioning and performing basic service configuration, if additional configuration on the service is required there are other providers that can be used to perform the task. 
```

Comment:
Suggested text:

[Aiven Terraform Provider](https://registry.terraform.io/providers/aiven/aiven/latest/docs) can be used to create and manage Aiven for PostgreSQL® service, PostgreSQL® databases, and users. If you need to perform additional configurations such as setting PostgreSQL default privileges, configure PostgreSQL publication, or reuse a PostgreSQL-based sub-module between different vendors to make the Terraform code homogeneous., you can consider using the [PostgreSQL® provider](https://registry.terraform.io/providers/cyrilgdn/postgresql/latest/docs).

Reply:
I used the markdown format for the links. Please remember to change to RST format.